src/main/java/uk/gov/digital/ho/hocs/casework/api/DashboardService.java

```diff
@@ -36,15 +36,21 @@ public DashboardService(DashboardSummaryFactory dashboardSummaryFactory,
 
     Map<UUID, Map<String, Integer>> getDashboard() {
         log.debug("Getting dashboard summary for user");
-        Set<UUID> allUserteams = userPermissionsService.getUserTeams();
-        if (allUserteams.isEmpty()) {
+
+        List<Summary> casesSummaries = new ArrayList<>();
+
+        Set<UUID> allUserTeams = userPermissionsService.getUserTeams();
+        if(!allUserTeams.isEmpty()) {
+            casesSummaries.addAll(summaryRepository.findTeamsAndCaseCountByTeamUuid(allUserTeams));
+        } else {
             log.warn("No teams - Returning 0 Stages", value(EVENT, TEAMS_STAGE_LIST_EMPTY));
             return Collections.emptyMap();
         }
 
         Set<String> caseTypes = userPermissionsService.getCaseTypesIfUserTeamIsCaseTypeAdmin();
-
-        List<Summary> casesSummaries = summaryRepository.findTeamsAndCaseCountByTeamUuidandCaseTypes(allUserteams, caseTypes);
+        if(!caseTypes.isEmpty() && (caseTypes.size() == 1 && !caseTypes.contains(""))) {
```

Comment:
I didn't want to change the existing code, but its because the caseTypes returns setOf("") instead of an empty set. 

We've had a call and agreed we can remove the odd behaviour in findTeamsAndCaseCountByTeamUuidandCaseTypes()